.github/workflows/test.yml

```diff
@@ -7,14 +7,12 @@ jobs:
       fail-fast: false
       matrix:
         os:
-        - ubuntu-20.04
-        - macos-11
-        # Build binaries on older systems to require lower versions of libc and
-        # other system libraries.
```

Comment:
We can't do this in the binary-building runs -- see the comment that this commit is proposing to remove!

I'm going to cherry-pick parts of this PR (with credit) into a new history, and merge that. Thank you!